sympy/combinatorics/tests/test_homomorphisms.py

```diff
@@ -101,5 +103,7 @@ def test_find_homomorphism():
     # Two groups of the same prime order are isomorphic to each other.
     G = FpGroup(F, [a, b**5])
     H = CyclicGroup(5)
+    list_hom = find_homomorphism(G, H, surjective=True, all=True)
+    assert all(elem.is_injective() for elem in list_hom)
```

Comment:
It would be better if `G` and `H` weren't isomorphic so that we could test just the surjections.